x/swagger/swagger_types_global.go

```diff
@@ -23,10 +20,7 @@ type ErrorGeneric struct {
 // swagger:model genericError
 type GenericError struct{ herodot.DefaultError }
 
-// Empty responses are sent when, for example, resources are deleted. The HTTP status code for empty responses is typically 201.
+// Empty responses are sent when, for example, resources are deleted. The HTTP status code for empty responses is typically 204.
 //
 // swagger:response emptyResponse
-//
-//nolint:deadcode,unused
-//lint:ignore U1000 Used to generate Swagger and OpenAPI definitions
-type emptyResponse struct{}
+type _ struct{}
```

Comment:
unaddressable names don't work with go-swagger afaik, but I might be wrong (or it has been fixed). Easiest way is to run make sdk and check the openapi diff

Reply:
Works!